### PR TITLE
Feature/new stacks

### LINKED ISSUE #1077: Add new stacks to docs
**Description**

docs.docksal.io does not have references to newer stacks

** Resolution **

Update docs to contain new stacks: Pantheon and Default-nodb

### PR DESCRIPTION
**Pull request by JDDoesDev**
This PR adds pantheon and default-nodb to the list of available stacks.  Closes #1077

### CODE REVIEW

File: docs/content/stack/zero-configuration.md [hunk 1]
```diff
@@ -25,5 +25,7 @@ DOCKSAL_STACK="acquia"
 The following stacks are available:
 
 - `default` - web, db, cli (assumed, when none specified)
+- `default-nodb` - web, cli
 - `acquia` - web, db, cli, varnish, memcached, solr (used specifically for [Acquia](https://www.acquia.com/) hosted projects)
+- `pantheon` - web (Nginx), db (MariaDB), cli, varnish, solr (used specifically for [Pantheon](https://www.pantheon.io/) hosted projects)
```
Comment (lmakarov) [inline docs/content/stack/zero-configuration.md]: Missed redis

```suggestion
- `pantheon` - web (Nginx), db (MariaDB), cli, varnish, redis, solr (used specifically for [Pantheon](https://www.pantheon.io/) hosted projects)
```

File: docs/content/stack/zero-configuration.md [hunk 2]
```diff
@@ -25,5 +25,7 @@ DOCKSAL_STACK="acquia"
 The following stacks are available:
 
 - `default` - web, db, cli (assumed, when none specified)
+- `default-nodb` - web, cli
 - `acquia` - web, db, cli, varnish, memcached, solr (used specifically for [Acquia](https://www.acquia.com/) hosted projects)
```
Comment (lmakarov) [inline docs/content/stack/zero-configuration.md]: Let's add the same details for the acquia stack.

```suggestion
 - `acquia` - web (Apache), db (MySQL), cli, varnish, memcached, solr (used specifically for [Acquia](https://www.acquia.com/) hosted projects)
```

Comment (JDDoesDev) [reply docs/content/stack/zero-configuration.md]: I love this idea!

**Comment by JDDoesDev** [comment]
I've also added the descriptions to the other stacks where appropriate.

**Comment by lmakarov** [comment]
Thanks!

**Pull request merged**